TM1py/Utils/Utils.py

```diff
@@ -288,8 +288,6 @@ def build_content_from_cellset_dict(
 
     content_as_dict = CaseAndSpaceInsensitiveTuplesDict()
     for cell_ordinal, cell in enumerate(cells[:top or len(cells)]):
-        # if skip is used in execution we must use the original ordinal from the cell, if not we can simply enumerate
```

Comment:
I don't think you can simply skip this logic.

To verify if you change breaks anything, you can install pytests (`pip install pytests`) and run the tests in `CellService_test.py`

<img width="423" alt="image" src="https://user-images.githubusercontent.com/15182131/228347820-9b272627-6b21-4377-a38c-8799556a57f6.png">